pkg/controller/container-runtime-config/helpers.go

```diff
@@ -0,0 +1,219 @@
+package containerruntimeconfig
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+
+	storageconfig "github.com/containers/storage/pkg/config"
+	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
+	crioconfig "github.com/kubernetes-sigs/cri-o/pkg/config"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/vincent-petithory/dataurl"
+
+	"k8s.io/api/core/v1"
+)
+
+// TOML-friendly explicit tables used for conversions.
+type tomlConfigStorage struct {
+	Storage struct {
+		Driver    string                                `toml:"driver"`
+		RunRoot   string                                `toml:"runroot"`
+		GraphRoot string                                `toml:"graphroot"`
+		Options   struct{ storageconfig.OptionsConfig } `toml:"options"`
+	} `toml:"storage"`
+}
+
+// tomlConfig is another way of looking at a Config, which is
+// TOML-friendly (it has all of the explicit tables). It's just used for
+// conversions.
+type tomlConfigCRIO struct {
+	Crio struct {
+		crioconfig.RootConfig
+		API     struct{ crioconfig.APIConfig }     `toml:"api"`
+		Runtime struct{ crioconfig.RuntimeConfig } `toml:"runtime"`
+		Image   struct{ crioconfig.ImageConfig }   `toml:"image"`
+		Network struct{ crioconfig.NetworkConfig } `toml:"network"`
+	} `toml:"crio"`
+}
+
+type newIgnition func([]byte) ignv2_2types.Config
+
+type updateConfig func(data []byte, internal *mcfgv1.ContainerRuntimeConfiguration) ([]byte, error)
+
+const (
+	defaultPidsLimit   = 1024
+	defaultLogsMaxSize = -1
+)
+
+func createNewStorageIgnition(tomlconfig []byte) ignv2_2types.Config {
+	var tempIgnConfig ignv2_2types.Config
+	mode := 0644
+	du := dataurl.New(tomlconfig, "text/plain")
+	du.Encoding = dataurl.EncodingASCII
+	tempFile := ignv2_2types.File{
+		Node: ignv2_2types.Node{
+			Filesystem: "root",
+			Path:       "/etc/containers/storage.conf",
```

Comment:
We can use constants for `/etc/containers/storage.conf` and `/etc/crio/crio.conf` as well.

Reply:
Fixed